tests/foreman/ui/test_rhc.py

```diff
@@ -65,10 +66,18 @@ def fixture_setup_rhc_satellite(request, module_target_sat, module_rhc_org):
         module_target_sat.cli.Subscription.upload(
             {'file': manifests_path, 'organization-id': module_rhc_org.id}
         )
+        # Enable and sync required repos
+        repo1_id = enable_sync_redhat_repo(REPOS['rhel8_aps'], module_rhc_org.id)
+        repo2_id = enable_sync_redhat_repo(REPOS['rhel7'], module_rhc_org.id)
+        # Add repos to Content view
+        content_view = module_target_sat.api.ContentView(
```

Comment:
Dont you think the Content View name should come from settings file / dynaconf env var / CI Stage 2 ?